File: code/3e110.pas
program Bai110;
var a, b,c: real;
begin
  readln(a, b, c);
    if (a = b) and (b = c) then writeln(' DEU') 
    else
    if (a =b) or (a = c) or (b = c) then  write('CAN')
      else write('THUONG');
end.

File: code/3e110.cpp
#include <iostream>
using namespace std;

int main() 
{
    int a,b,c;
    cin>>a>>b>>c;
    if (a==b || a==c || c==b)
        cout<<"CAN";
    else if (a==b && b==c)
        cout<<"DEU";
    else cout<<"THUONG";
    return 0;
}